gazebo_plugins/src/gazebo_ros_moveit_planning_scene.cpp

```diff
@@ -401,16 +401,16 @@ void GazeboRosMoveItPlanningScene::UpdateCB()
                     //gzwarn<<"TRIANGLES"<<std::endl;
                     shape_msgs::Mesh mesh_msg;
                     mesh_msg.vertices.resize(n_vertices);
-                    mesh_msg.triangles.resize(n_vertices/3);
+                    mesh_msg.triangles.resize(n_vertices/3+1);
 
                     for(size_t v=0; v < n_vertices; v++) {
 
                       const int index = submesh->GetIndex(v);
                       const ignition::math::Vector3d vertex = submesh->Vertex(v);
 
-                      mesh_msg.vertices[index].x = vertex.X() * scale.X();
-                      mesh_msg.vertices[index].y = vertex.Y() * scale.Y();
-                      mesh_msg.vertices[index].z = vertex.Z() * scale.Z();
+                      mesh_msg.vertices[v].x = vertex.X() * scale.X();
+                      mesh_msg.vertices[v].y = vertex.Y() * scale.Y();
+                      mesh_msg.vertices[v].z = vertex.Z() * scale.Z();
```

Comment:
I'm not so sure about changing these lines

Reply:
Sorry, I haven't used these codes for a long time so I have no idea about it now. What should I do now? Should I resolve conversation or do other in github?